validate.lic

```diff
@@ -88,6 +88,14 @@ class DRYamlValidator
     echo("Valid skills are #{@valid_weapon_skills}")
   end
 
+  def assert_that_cycle_skills_named_properly(settings)
+    return unless settings.cycle_armors
+
+    settings.cycle_armors
+            .reject { |skill| ['Light Armor', 'Chain Armor', 'Brigandine', 'Plate Armor'].include?(skill) }
+            .each_key { |skill| error("Skill name in cycle_armors is not valid: #{skill}") }
```

Comment:
What do you think about checking that the values (i.e. items) are in the gear list? Too much?